### ISSUE TITLE
node-runner CLI invocation excludes cwd from module search path

### ISSUE THREAD

**Issue by moea** [cemerick/clojurescript.test#102]
Thanks for the project.

I'm not at all experienced with node, however with 0.12.5, `node /tmp/runner target` (the expansion of `:node-runner`) will only look inside `/tmp` for a `node_modules` directory, unless directed otherwise.  Would it be weird to do something with `NODE_PATH` in the middleware (e.g. include the Leiningen project directory)?  Or this something you'd expect users to configure themselves?

**Comment by cemerick** [comment]
I'm definitely not the person to ask what would be reasonable w.r.t. node. Can NODE_PATH not be changed at runtime (from within the script)? The middleware knows nothing of the runner scripts, and that should stay that way.

**Comment by moea** [comment]
Yeah, it looks like it can (module.paths is writable).  Sticking the cwd in there before loading anything seems like it would fix more issues than it'd create.  Let me know how you feel about a patch.

**Comment by vheuken** [comment]
I ran into the same issue as @moea, which I solved by adding `NODE_PATH=./node_modules` before `lein cljsbuild test`. That was simple enough. Then I ran into another issue. I have tests that require passing in files, so I'd have some test files in my project directory, but I couldn't access them because [Node's __dirname](https://nodejs.org/docs/latest/api/globals.html#globals_dirname) would be set to `/tmp` instead of the directory I called the test from. I know that I could just set another environment variable, but that's an annoying workaround. 

@cemerick Is there any reason why :node-runner uses /tmp as the directory? Just having it use the directory where `lein cljsbuild test` was called from would solve my issues and @moea's.
